### PR TITLE
System core info implementation

### PR DESCRIPTION
**Pull request by assafchamoy**

### CODE REVIEW

File: src/discrete-layer/components/system-status/system-core-info/system-core-info.dialog.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    overflow-y: hidden !important;
+    overflow-x: hidden;
+  }
+  
+  #systemCoreInfoDialog .mdc-dialog {
+    justify-content: center;
+  }
+  
+  #systemCoreInfoDialog #systemCoreInfoBody {
+    --tab-bar-height: 50px;
```
Comment (alebinson) [inline src/discrete-layer/components/system-status/system-core-info/system-core-info.dialog.css:14]: move it on inline style level
<DialogContent id="systemCoreInfoBody" **style="--tab-bar-height: 50px"**>

**Review by alebinson — APPROVED** [review]
gr8!

File: src/discrete-layer/components/system-status/system-core-info/internal-service/internal-service.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+interface InternalServiceProps {
+  service: DeploymentWithServicesModelType;
+}
+
+interface Services {
+  name: string;
+  uid: string;
+  addresses: string[];
+}
+const THEME_SUCCESS_COLOR = 'var(--mdc-theme-gc-success)';
```
[the remark this answers is not in the document]
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/system-core-info/internal-service/internal-service.tsx:18]: fixed

File: src/discrete-layer/components/system-status/system-core-info/system-core-info.dialog.css (hunk 1 above)
Comment (assafchamoy) [reply src/discrete-layer/components/system-status/system-core-info/system-core-info.dialog.css:14]: fixed

File: src/common/i18n/en.json [hunk 3]
```diff
@@ -386,6 +387,17 @@
   "system-status.task.fields.created.label": "Created",
   "system-status.task.fields.updated.label": "Updated",
 
+  "system-core-info.title": "System Info",
+  "system-core-info.innerServicesTitle": "Internal Services",
+  "system-core-info.externalServicesTitle": "External Services",
+  "system-core-info.versionText" :"Version",
+  "system-core-info.servicesText":"Services",
+  "pycsw.raster.url": "raster",
```
[the remark this answers is not in the document]
Comment (assafchamoy) [reply src/common/i18n/en.json:395]: Fixed

**Pull request merged**